source/includes/_endpoint-deck.md

```diff
@@ -0,0 +1,413 @@
+  "graph": [
+    "https://beta.crunch.io/api/datasets/223fd4/decks/1/",
+    "https://beta.crunch.io/api/datasets/223fd4/decks/2/",
+    "https://beta.crunch.io/api/datasets/223fd4/decks/3/"
+  ]
+}
+
+```
+
+#### PATCH
+
+PATCH the order resource to change the order of the decks. A 204 response indicates success.
+
+If the PATCH payload contains only a subset of available decks, those
+decks not referenced will be appended at the bottom of the top level graph in arbitrary
+order.
+
+```json
+{
+  "element": "shoji:order",
+  "self": "https://beta.crunch.io/api/datasets/223fd4/decks/order/",
+  "graph": [
+    "https://beta.crunch.io/api/datasets/223fd4/decks/1/",
+    "https://beta.crunch.io/api/datasets/223fd4/decks/3/"
+  ]
+}
+```
+
+Including invalid URLs or URLs to decks that are not present in the catalog will
+ return a 400 response from the server.
+
+The deck order should always be a flat list of URLs. Nesting or grouping is not
+supported by the web application.
+
+## Slides
+
+Each deck contains a list of slides, in which analyses are saved. They
+are available through the slides catalog.
+
+### Catalog
+
+`/datasets/{id}/decks/{deck_id}/slides/`
+
+#### GET
+
+Returns a `shoji:catalog` with the slides for this deck.
+
+```json
+
+{
+    "element": "shoji:catalog",
+    "self": "/api/datasets/123/decks/123/slides/",
+    "orders": {
+        "flat": "/api/datasets/123/decks/123/slides/flat/"
+    },
+    "specification": "https://beta.crunch.io/api/specifications/slides/",
+    "description": "A catalog of the Slides in this Deck",
+    "index": {
+        "/api/datasets/123/decks/123/slides/123/": {
+            "analysis_url": "/api/datasets/123/decks/123/slides/123/analyses/123/",
+            "subtitle": "z",
+            "display": {
+                "value": "table"
+            },
+            "title": "slide 1"
+        },
+        "/api/datasets/123/decks/123/slides/456/": {
+            "analysis_url": "/api/datasets/123/decks/123/slides/456/",
+            "subtitle": "",
+            "display": {
+                "value": "table"
+            },
+            "title": "slide 2"
+        }
+    },
+    "template": "{\"query\": {\"measures\": \"Object with keyed measures functions\", \"dimensions\": \"Array of variable references\", \"weight\": \"Optional weight variable URL\"}, \"display_settings\": {}, \"query_environment\": {\"filter\": \"Array of filter urls\"}}"
+}
+
+```
+
+Each tuple on the slides catalog contains the following keys:
+
+Name | Type | Description
+---- | ---- | -----------
+analysis_url | url | Points to the analysis contained on this slide
+title | string | Optional title for the slide
+subtitle | string | Optional subtitle for the slide
+display | object | Stores settings used to load the analysis
+
+#### POST
+
+To create a new slide an analysis has to be posted to the catalog. The payload, described in the [below](#analyses) section, should be
```

Comment:
Yes, slides contain an analysis catalog, _but_ since we're only supporting one per slide, the slides exposes a direct access to it to save client's the extra jumps.

A slide entity contains title, subtitle. You cannot currently create a bare slide by POSTing that alone. You post this janus-slide that has an analysis +title +subtitle and creates the slide and the first(only) analysis. There is no API to add more analysis to it.